color-eyre/tests/data/theme_panic_control_no_spantrace.txt

```diff
@@ -1,18 +1,18 @@
-    Finished dev [unoptimized + debuginfo] target(s) in 0.02s
-     Running `target/debug/examples/theme_test_helper`
+    Finished dev [unoptimized + debuginfo] target(s) in 0.07s
+     Running `/home/freja/dev/rust/eyre/target/debug/examples/theme_test_helper`
```

Comment:
Does this hardcode your home directory into the UI test, meaning only a machine with your username on it can run the test successfully?

Reply:
No, it only checks for the ansi escape codes. If you look in the other files it contains Jane's home directory

Reply:
Fixed with a bit of `%s/freja/username/g`ing